### DISCUSSION TITLE
loading a satellite from OME

### DISCUSSION THREAD

**Discussion by ggembafzj** [skyfielders/python-skyfield#978] [Q&A] [answered]
Hey is there a function to load a satellite from OME data just using position and velocity?
I have found an application for OMM data but have not found any implementation using just position and velocity and setting the satellite object to this.
Am using skyfield for orbit and attitude prediction and assessment of a small satellite mission and would love not having to buy an STK license.
Thanks!

**Comment by gfireman** [comment]
I have a similar need, using OEM ephemeris data.

**Comment by brandon-rhodes** [comment, marked as answer]
No, Skyfield does not currently have a routine for that. You can follow a bit of discussion here, though, about the issues around loading a satellite from a list of positions:

https://github.com/skyfielders/python-skyfield/issues/457
